### ISSUE TITLE
'break' on non-multiple of 8 offsets leaves garbage data

### ISSUE THREAD

**Issue by amj** [TheKitty/Untz_Monome#2]
according to the spec as i understand it, the bulk-update commands that operate on 8x8 blocks can pass along x,y offsets that aren't actually multiples of 8.  Right now, untz_monome will break if it sees those, leaving the remaining serial bytes specifying the data left in the buffer.

So, any 'break' in that switch statement can potentially leave junk in the incoming serial stream to foul up the next command.

this is based on my reading of http://monome.org/docs/serial.txt, which seems a little obtuse, but hey.

**Comment by TheKitty** [comment]
I tried to decipher the functions from multiple documents. The company
doesn't have much incentive for people to make home grown sequencers. So
yes, until I get a working, communicating version, testing each function
will be very hard unless the docs are clear

**Comment by amj** [comment]
Yeah, i'll try patching serialosc to give me some logging to find out what's going on!  I got a reply from tehn in the forum though, so i'm hopeful! :)

**Comment by TheKitty** [comment]
Awesome, great work
